src/components/styles/PageSection.styled.ts

```diff
@@ -1,12 +1,25 @@
-import styled from 'styled-components'
+import styled, { keyframes } from 'styled-components'
 import Null from '../../assets/noimg.png'
 
+const fadeinuup = keyframes`
+    from {
+      opacity: 0;
+      transform: translateY(100%);
+    }
+    to {
+      opacity: 1;
+      transform: none;
+    }
+`
+
 export const MainContainer = styled.div`
   height: 33%;
   width: 80%;
   display: flex;
   margin: auto;
   background-color: black;
+  animation-duration: 1s;
```

Comment:
consider making this optional - use props to make it animate, also discuss w the person doing events page to see how they did it